src/frequenz/sdk/microgrid/component/_component.py

```diff
@@ -67,16 +74,32 @@ class ComponentCategory(Enum):
     """Possible types of microgrid component."""
 
     NONE = components_pb.ComponentCategory.COMPONENT_CATEGORY_UNSPECIFIED
+    """Unspecified component category."""
+
     GRID = components_pb.ComponentCategory.COMPONENT_CATEGORY_GRID
+    """Grid component."""
+
     METER = components_pb.ComponentCategory.COMPONENT_CATEGORY_METER
+    """Meter component."""
+
     INVERTER = components_pb.ComponentCategory.COMPONENT_CATEGORY_INVERTER
+    """Inverter component."""
+
     BATTERY = components_pb.ComponentCategory.COMPONENT_CATEGORY_BATTERY
+    """Battery component."""
+
     EV_CHARGER = components_pb.ComponentCategory.COMPONENT_CATEGORY_EV_CHARGER
+    """EV charger component."""
+
     CHP = components_pb.ComponentCategory.COMPONENT_CATEGORY_CHP
+    """CHP component."""
 
-    # types not yet supported by the API but which can be inferred
-    # from available graph info
     PV_ARRAY = 1000001
+    """PV array component.
+
+    Types not yet supported by the API but which can be inferred from available
+    graph info.
```

Comment:
This sounds like an implementation detail that should be kept as a code comment rather than a docstring. @shsms ?

Reply:
I don't think this `PV_ARRAY` component should remain here at all.  We don't use it anymore.  This is from the old `MicrogridData` days.

Reply:
will revert it

Reply:
> I don't think this PV_ARRAY component should remain here at all. We don't use it anymore. This is from the old MicrogridData days.

Then I'll add a new commit to remove it if that's the case

Reply:
I've removed PV_ARRAY. Would this need an entry into the release notes?